app/controllers/application_controller.rb

```diff
@@ -1,4 +1,16 @@
 # frozen_string_literal: true
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def append_info_to_payload(payload)
+    super
+    payload[:user_id] = current_user.try(:id)
```

Comment:
`try(:current_user).try(:id)`, as you might not have this method defined

Same for organization